interceptors/CrmDataInterceptor.cfc

```diff
@@ -42,18 +42,6 @@ component {
         }
     }
 
-    private void function _maybeEnsureUniquePrimaryAddress( interceptData ) {
```

Comment:
It's not clear why this function has been removed - it is still in use. Looks not correct to me.

Reply:
The function existed twice. I only removed the duplicate.